trino/integration_test.go

```diff
@@ -110,6 +112,14 @@ func TestMain(m *testing.M) {
 					"8080/tcp",
 					"8443/tcp",
 				},
+			}, func(hc *docker.HostConfig) {
+				hc.Ulimits = []docker.ULimit{
+					{
+						Name: "nofile",
+						Hard: 4096,
+						Soft: 4096,
+					},
+				}
```

Comment:
Trino requires a minimum of 4096 descriptors in order to startup. On my linux machine, I found docker defaults to a limit of 1024.

I've updated the test configuration such that we can be sure the container is configured with reasonable ulimits.